pyproject.toml

```diff
@@ -33,4 +34,4 @@ package-dir = {"" = "src"}
 packages = {find = {where = ["src", "nodes"]}}
 
 [tool.setuptools.dynamic]
-dependencies = {file = ["requirements.txt"]}
+dependencies = {file = ["requirements.txt"]}
```

Comment:
We now still have redundancy right? Both pyproject.toml and requirements. Wondering if we can eventually move to one source of truth.

Reply:
Yes, I tested removing requirements.txt here https://github.com/livepeer/comfystream/commit/74297aaf5eda8d31bb2f240cdcf618df7a80f599 and it works fine with simple `pip install`. There may be reasons to keep requirements.txt, but we do not specifically need it and it can also be regenerated using setuptools. I decided to keep it for now to keep the change concise, but will defer to your decision